db/schema.rb

```diff
@@ -10,7 +10,20 @@
 #
```

Comment:
should not be added in PR

Reply:
schema.rb and database.yml files are included in .gitignore. But still they push.